entrypoint.sh

```diff
@@ -30,4 +30,11 @@ else
   fi
 fi
 
-aws s3 cp "$FILENAME".sql.gz s3://"$INPUT_AWS_S3_BUCKET"
+if [ -z "${INPUT_AWS_S3_DIRECTORY}" ] ]
```

Comment:
Not familiar enough with how `aws_s3_directory` in config is translated into environment var INPUT_AWS_S3_DIRECTORY, but if that is how it works, then the code looks sane and clean.

Reply:
I forget where I've found it before and a quick search didn't turn much up, but I'm following the pattern used elsewhere in this repo.

Good eye on the naming difference though. 🦅

Reply:
If you want to use this, can you point Composer to this branch to give this a spin, then once we confirm it works for you, we can merge it?